misk-testing/src/test/kotlin/misk/service/ServiceTestingModuleTest.kt

```diff
@@ -0,0 +1,108 @@
+package misk.service
+
+import com.google.common.util.concurrent.AbstractIdleService
+import com.google.common.util.concurrent.Service
+import com.google.common.util.concurrent.ServiceManager
+import com.google.inject.Guice
+import com.google.inject.Key
+import misk.DependentService
+import misk.MiskServiceModule
+import misk.inject.KAbstractModule
+import misk.inject.getInstance
+import misk.inject.keyOf
+import org.junit.jupiter.api.Test
+import java.util.concurrent.TimeUnit
+import java.util.concurrent.atomic.AtomicBoolean
+import javax.inject.Inject
+import javax.inject.Singleton
+
+internal class ServiceTestingModuleTest {
+  @Test fun serviceTestModuleAddsTestSpecificDependency() {
+    val injector = Guice.createInjector(object : KAbstractModule() {
+      override fun configure() {
+        install(MiskServiceModule())
+        multibind<Service>().to<LowestLevelService>()
+        install(ServiceTestingModule.withExtraDependencies<AppService>(
+            keyOf<ExternalServiceSpinupService>())
```

Comment:
are we exercising any of the "extra dependencies" bit? Is that worthwhile?

Reply:
Not sure what you mean? This test confirms the extra dependencies

Reply:
Derp, misread the constructor. 👍